3_harmonize/src/harmonize_sdd.R

```diff
@@ -473,7 +473,7 @@ harmonize_sdd <- function(raw_sdd, p_codes){
   print(
     paste(
       round((nrow(sdd_bottom_depth)) / nrow(converted_units_sdd) * 100, 3),
-      "% of samples have no reported ActivityBottomDepthHeightMeasure.MeasureValue that can be 
+      "% of samples have no reported ActivityBottomDepthHeightMeasure.MeasureValu. These will be 
```

Comment:
```suggestion
      "% of samples have no reported ActivityBottomDepthHeightMeasure.MeasureValue but Secchi on bottom has been indicated and a harmonized value is present. These will be 
```
There are additional conditions to describe the % of samples here. Added more context to add readabillity.